HIP/hip-0000-FT-JSON-SCHEMA.md

```diff
@@ -0,0 +1,181 @@
+---
+hip: 0000
+title: Fungible Token Metadata JSON Schema
+author: Paul Madsen, <paul@calaxy.com>, Cooper Kunz, <cooper@calaxy.com>
+needs-council-approval: No
+status: Active
```

Comment:
I believe the status should be set to either 'Review' or 'Draft' at this stage of the HIP